app/forms/openid_connect_authorize_form.rb

```diff
@@ -113,6 +114,12 @@ def validate_unauthorized_scope
     errors.add(:scope, t('openid_connect.authorization.errors.unauthorized_scope'))
   end
 
+  def validate_prompt
```

Comment:
Should this add an error if the `prompt` param is nil? It looks like it will.

The validation for `prompt` on line 32 looks like it validates presence, but also looks like it has `allow_nil: true`. That is confusing to me. If we don't allow it to be blank is there any reason to not to pull the `allow_nil` out of that validation to clean it up.

Reply:
I looked around and don't see any reason to allow prompt to be nil... tests are still passing without it... Looks better now?